.github/workflows/test.yaml

```diff
@@ -11,16 +11,9 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - uses: actions/cache@v2
-        with:
-          path: |
-            ~/.cargo/registry
-            ~/.cargo/git
-            target
-          key: ${{ runner.os }}-build-android-${{ hashFiles('**/Cargo.lock') }}
-          restore-keys: |
-            ${{ runner.os }}-build-android
+      - uses: Swatinem/rust-cache@v1
```

Comment:
What is the advantage of rust-cache here? Reading its github-page I'm not sure if it also caches the compiled files or just the downloaded ones.

Reply:
It is more readable and better caches than `actions/cache`. It reduce [greatly the test time](https://github.com/c4dt/arti-rest/runs/2885756369).

Reply:
I think the other cache only saved when the workflow passed or something like that. It already reduced the test time a lot, but only after a successful test.